app/channels/application_cable/channel.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
```

Comment:
Can you please remove this comments in all containing files, if this is not necessary. Thanks!